src/lib.rs

```diff
@@ -54,8 +59,8 @@ pub unsafe fn init(ty: ApplicationType) -> Result<Context, InitError> {
     if INITIALIZED.swap(true, Ordering::Acquire) {
         panic!("OpenVR has already been initialized!");
     }
-
-    Ok(Context { live: AtomicBool::new(true) })
+    INITIALIZED.store(true,Ordering::Release);
```

Comment:
I need to double check the behavior of this.

We recently had to patch the initialization atomic internal to the library because it would go out of sync with the fact that OpenVR was or was not properly initialized (making it impossible to run a program if there was a init error. e.g: a background application trying to start when SteamVR was not running. You are supposed to be able try again. There was a bug there for example)

Reply:
All I want to make sure of is that you can try to init SteamVR, it fails for reasons that are extenral to the program, and you can _try again_ multiple times up until SteamVR is actually ready for you to work 

Background applications will not automatically start SteamVR for example, and will return with error from the OpenVR init.

This library used to not support that properly. It currently does though. :slightly_smiling_face: